python/lsst/sconsUtils/builders.py

```diff
@@ -573,15 +573,15 @@ def makeVersionModule(target, source, env):
             outFile.write("# -------- This file is automatically generated by LSST's sconsUtils -------- #\n")
 
             what = "__version__"
-            outFile.write("%s = '%s'\n" % (what, version))
```

Comment:
I'm fine with the single quotes being switched to the outer and the double quotes then not needing to be escaped.

f-strings can also be used these days.

Reply:
Okay, done.